.github/workflows/ci-label.yml

```diff
@@ -55,6 +55,6 @@ jobs:
           labels: 'type: performance'
 
       - uses: actions-ecosystem/action-add-labels@v1
-        if: startsWith(github.event.pull_request.head.ref, 'pre-commit-ci-update-config') || startsWith(github.event.pull_request.head.ref, 'conda-lock-auto-update')
+        if: startsWith(github.event.pull_request.head.ref, 'pre-commit-ci-update-config') || startsWith(github.event.pull_request.head.ref, 'conda-lock-auto-update') || startsWith(github.event.pull_request.head.ref, 'dependabot/github_actions/actions')
```

Comment:
It is better to use `github.actor` in my mind. See https://github.com/pyvista/pyvista/blob/6d1583bb51aecea964bde64107d8b2641c1f456b/.github/workflows/label.yml#L29 .

Reply:
Neat! Thanks @tkoyama010 :rocket: